integration/airflow/tests/integration/requests/secrets.json

```diff
@@ -0,0 +1,44 @@
+[{
+    "eventType": "START",
+    "job": {
+        "name": "secrets.secrets",
+        "namespace": "food_delivery"
+    },
+    "run": {
+        "facets": {
+            "airflow_version": {
+                "taskInfo": "{{ not_match(result, '1500100900') }}"
+            },
+            "unknownSourceAttribute": {
+                "unknownItems": [
+                    {
+                        "name": "SecretsOperator",
+                        "properties": {
+                            "password": "***"
+                        }
```

Comment:
This test is fine but does not make us 100% sure that all secrets have been redacted from properties of the operator. This could be checked somewhere in unit test if that's easier to do?